client/.editorconfig

```diff
@@ -0,0 +1,18 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
Could this be part of the real root config?

Reply:
same for .gitattributes, .gitignore, ...

Reply:
Of course, I just generated all to see what will be used and as first part separated from the main source.